app/src/wavevr/cpp/DeviceDelegateWaveVR.cpp

```diff
@@ -373,13 +422,20 @@ void
 DeviceDelegateWaveVR::SetClearColor(const vrb::Color& aColor) {
   m.clearColor = aColor;
 }
+
 void
 DeviceDelegateWaveVR::SetClipPlanes(const float aNear, const float aFar) {
   m.near = aNear;
   m.far = aFar;
   m.InitializeCameras();
 }
 
+void
+DeviceDelegateWaveVR::SetFoveatedLevel(const int32_t aAppLevel, const int32_t aWebVRLevel) {
```

Comment:
It doesn't make any sense to pass the WebVR level into the DeviceDelegate. We should probably just remove it from the function call. When it is implemented it will either have to to gecko via the external VR interface.

Reply:
I just removed the variable for setting WebVR level. I think we would to send the WebVR level from FxR settings to GV via the external VR interface to make WebGL framebuffer to change its foveated level.